src/state/user/hooks/index.tsx

```diff
@@ -418,6 +418,9 @@ export function toV2LiquidityToken([tokenA, tokenB]: [Token, Token]): Token {
 export function useTrackedTokenPairs(): [Token, Token][] {
   const { chainId } = useActiveWeb3React()
   const tokens = useAllTokens(true)
+  const filterTokens = Object.keys(tokens)
+    .slice(0, 1000)
```

Comment:
Note that this is intermediate solution, we will fetch too many pairs if token list is too large